vent/common/message.py

```diff
@@ -11,20 +13,34 @@ def __init__(self, name, value, timestamp, loop_counter):
 
 
 class SensorValues:
-    def __init__(self, pip=None, peep=None, fio2=None, temp=None, humidity=None, pressure=None, vte=None, breaths_per_minute=None,
-                 inspiration_time_sec=None, timestamp=None, loop_counter = None):
-        self.pip = pip
-        self.peep = peep
-        self.fio2 = fio2
-        self.temp = temp
-        self.humidity = humidity
-        self.pressure = pressure
-        self.vte = vte
-        self.breaths_per_minute = breaths_per_minute
-        self.inspiration_time_sec = inspiration_time_sec
+    def __init__(self, timestamp=None, loop_counter=None, **kwargs):
```

Comment:
this is sorta a chimerical class because i'm trying to make this consistent with what we already had but it's not necessarily how i would do it if it were up to me. As written it takes the string representations of the `ValueName` enum and sets them as attributes. it could also store a dict of `{ValueName: value}` pairs, but i just didn't like the split-implementation situation we had before where I was suddenly being given a `dict` and then in the controller the `SensorValues` object was being created with a bunch of hardcoded variable names that were inconsistent with `ValueName`.

Reply:
I'm fine with both options. Maybe we can talk this later.